.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.12.5
+      - image: circleci/golang:1.12.7
```

Comment:
Unrelated to the PR subject. I saw some unexplicable flaky behaviour in test runs and tried a remedy.